### ISSUE TITLE
Missing step to S3EventLambda	

### ISSUE THREAD

**Issue by juanzapa** [aws-samples/cases-with-contact-lens-genai-summary#1]
I'm not sure if I haven't followed the guide correctly but I think the step is missing where the Lambda trigger called S3EventLambda is configured. In my case, by following the solution step by step, I have managed to display the complete stack and the contact-case information is being saved in Dynamo, but the case comment does not appear yet. Reviewing the guide and the code, I see that the lambda S3EventLambda is in charge of doing this but does not have any trigger configured to be activated. Can you tell me if I am missing any steps that I have not read in the guide or if I definitely need to add them?
<img width="821" alt="Captura de Pantalla 2024-06-04 a la(s) 9 26 31 p m" src="https://github.com/aws-samples/cases-with-contact-lens-genai-summary/assets/57536157/7683d64c-7a43-49bc-b163-8296edc26610">

Thank you so much

**Comment by juanzapa** [comment]
Now I think the problem is that the ReadMe guide refers to the fact that the trigger should be connected to the CaseEventFunction lambda when it really should be connected to the S3EventFunction

**Comment by reicolina** [comment]
Yes. I had the same problem... and opened this PR to update the doc 📝  https://github.com/aws-samples/cases-with-contact-lens-genai-summary/pull/2
